test/MessageSenderTest.cpp

```diff
@@ -96,7 +98,7 @@ ASYNC_TEST(MessageSender, testGetQueryIdGetterWorks) {
   QueryHub queryHub{ioContext};
 
   {
-    auto messageSender = MessageSender::create(std::move(queryId), queryHub);
+    auto messageSender = MessageSender{std::move(queryId), queryHub};
```

Comment:
The `auto` is kind of redundant when placed next to an explicit call to the constructor. Same in line 63